filters/auth/oidc.go

```diff
@@ -481,7 +487,12 @@ func (f *tokenOidcFilter) Request(ctx filters.FilterContext) {
 		return
 	}
 
-	authorized(ctx, sub)
+	oidcInfoJson, err := json.Marshal(oidcInfo)
+	if err != nil {
+		unauthorized(ctx, sub, "failed to serialize oidc info", r.Host)
```

Comment:
just a hint: `sub` currently will stay uninitialized. (I see that the PR is in the "Draft" state.)